buildlib/azure-pipelines-release-drp.yml

```diff
@@ -73,6 +81,8 @@ stages:
               check_release_build $(Build.Reason) $(Build.SourceVersion) "AZP/DRP-RELEASE: "
             name: Result
             displayName: Check build condition
+            env:
+              AZURE_DEVOPS_EXT_PAT: $(AZURE_DEVOPS_EXT_PAT)
```

Comment:
why is this needed?

Reply:
Authentication is required for `az` CLI down the line.
The actual authentication token is stored securely in Azure and injected at runtime, without the risk of exposure in logs or the pipeline YAML.